src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const currentRent = 40;
+  const minDaysOff = 3;
+  const maxDaysOff = 7;
+  const minDiscountRent = 20;
+  const maxDiscountRent = 50;
+
+  if (days >= maxDaysOff) {
+    return currentRent * days - maxDiscountRent;
+  }
+
+  if (days >= minDaysOff) {
+    return currentRent * days - minDiscountRent;
+  }
+
+  return currentRent * days;
```

Comment:
The final return statement is clear and concise, correctly handling the case when no discounts apply.